wave_benchmark/include/wave/benchmark/pose_measurement.hpp

```diff
@@ -16,11 +16,11 @@ namespace wave {
  *  @{ */
 
 struct BenchmarkPose {
-    Rotation rotation;
+    RotationMd rotation;
```

Comment:
I suspect there is not much point in `BenchmarkPose` anymore. If the point was to combine a Rotation and Translation, wave_geometry's `RigidTransformMd` does that.

As long as behaviour is unchanged by this PR, can add a new issue to make that change.